users/templates/users/account_settings.html

```diff
@@ -1,165 +1,159 @@
+                                            <h4 class="fw-bolder fs-6">Content languages</h4>
+                                            <p class="text-light-emphasis">Add languages you’d like to see posts,
+                                                community recommendations, and other content in</p>
                                         </div>
                                         <div class="col d-flex justify-content-end">
-                                            <button type="button" class="btn btn-dark border-light rounded-pill">Connect to Apple</button>
-                                            <!-- Add logic for adding Apple account -->
+                                            <!-- Content languages field from UserSettingsForm -->
+                                            {{ form.content_lang }}
                                         </div>
                                     </div>
                                 </div>
-                                <div class="mb-3 mt-3">
-                                    <div class="row align-items-center">
-                                        <div class="col d-flex flex-column">
-                                            <h4 class="fw-bolder fs-6">Connect to Google</h4>
-                                            <p class="text-light-emphasis fs-6">Connect account to log in to Reddit with Google</p>
+
+                                <!-- Location customization from form -->
+                                <div class="mb-2">
+                                    <div class="d-flex flex-column flex-fill">
+                                        <h4 class="fw-bolder fs-6">Location customization</h4>
+                                        <p class="text-light-emphasis mb-0">Specify a location to customize your
+                                            recommendations and feed. Reddit does not track your precise geolocation
+                                            data.</p>
+                                        <div class="col-md-6">
+                                            <!-- Location field from UserSettingsForm -->
+                                            {{ form.location }}
                                         </div>
-                                        <div class="col d-flex justify-content-end">
-                                            <button type="button" class="btn btn-dark border-light rounded-pill">Connect to Apple</button>
-                                            <!-- Add logic for adding Google account account -->
+                                    </div>
+                                </div>
+                                <!-- Opt into is over 18 from form -->
+                                <div class="mb-2">
+                                    <div class="d-flex flex-column flex-fill">
+                                        <p></p>
```

Comment:
Please do not use empty paragraphs for styling.

Reply:
Removed